cli/tests/test_next_prev_commands.rs

```diff
@@ -567,6 +567,114 @@ fn test_next_editing() {
     "###);
 }
 
+#[test]
+fn test_prev_conflict() {
+    // Make the first commit our new parent.
+    let test_env = TestEnvironment::default();
+    test_env.jj_cmd_ok(test_env.env_root(), &["init", "repo", "--git"]);
+    let repo_path = test_env.env_root().join("repo");
+    let file_path = repo_path.join("content.txt");
+    test_env.jj_cmd_ok(&repo_path, &["commit", "-m", "first"]);
+    std::fs::write(&file_path, "second").unwrap();
+    test_env.jj_cmd_ok(&repo_path, &["commit", "-m", "second"]);
+    test_env.jj_cmd_ok(&repo_path, &["commit", "-m", "third"]);
+    // Create a conflict in the first commit, where we'll jump to.
+    test_env.jj_cmd_ok(&repo_path, &["edit", "description(first)"]);
+    std::fs::write(&file_path, "first").unwrap();
+    test_env.jj_cmd_ok(&repo_path, &["new", "description(third)"]);
+    test_env.jj_cmd_ok(&repo_path, &["commit", "-m", "fourth"]);
+    let (stdout, stderr) = test_env.jj_cmd_ok(&repo_path, &["prev", "--conflict"]);
+    // We now should be a child of `second`.
+    insta::assert_snapshot!(stdout, @"");
+    insta::assert_snapshot!(stderr, @r###"
+    Working copy now at: vruxwmqv a225a03d (conflict) (empty) (no description set)
+    Parent commit      : royxmykx a8d7fcf4 (conflict) (empty) fourth
```

Comment:
The description doesn't match the output (I think this is the same place as where it was `second` vs `third` before)?)

Reply:
that's correct but related to our conversation in https://github.com/martinvonz/jj/pull/3248#discussion_r1614021777, so I'm going to wait until that is resolved to either keep or adjust the comment (`roots()` => second, `heads()` => fourth).

Reply:
I've adjusted the comment to be `fourth`.